pkg/liquidity-mining/contracts/VotingEscrowRemapper.sol

```diff
@@ -205,8 +199,8 @@ contract VotingEscrowRemapper is IVotingEscrowRemapper, SingletonAuthentication,
     function clearNetworkRemapping(address localUser, uint16 chainId) external payable override nonReentrant {
         require(localUser != address(0), "localUser cannot be zero address");
         require(!_isAllowedContract(localUser) || localUser == msg.sender, "localUser is still in good standing");
-        IOmniVotingEscrow omniVotingEscrow = getOmniVotingEscrow();
-        require(address(omniVotingEscrow) != address(0), "Omni voting escrow not set");
+        IOmniVotingEscrowAdaptor omniVotingEscrowAdaptor = getOmniVotingEscrowAdaptor();
+        require(address(omniVotingEscrowAdaptor.getOmniVotingEscrow()) != address(0), "Omni voting escrow not set");
```

Comment:
With the separation of concerns in https://github.com/balancer/balancer-v2-monorepo/pull/2463/commits/e70e32dabd25890f48eb8b93e702207373c14d9e, `getOmniVotingEscrow` is out of the settings just because of this.

This is just a small gas optimization in the end. I could move the require to `sendUserBalance` and `estimateSendUserBalance` inside the adaptor, but that would mean that we would be checking if the voting escrow is set more than once in clear / set network remapping.

WDYT?